src/main/java/frc/robot/Constants.java

```diff
@@ -600,4 +600,46 @@ public static final class CraneConstants {
     public static final Translation2d kPositionIntake =
       new Translation2d(Units.degreesToRadians(32.042), 0.6824);
   }
+
+  public static final class ClimberConstants {
+    public static final boolean kEnable = false;
+    public static final int kMotorID = 16;
+    public static final int kEncoderChannelA = 2;
```

Comment:
This is called `kEncoderChannelAbs`. The through-bore encoder has A, B, I, and ABS.

https://docs.revrobotics.com/rev-crossover-products/sensors/tbe/application-examples